network-mux/src/Network/Mux.hs

```diff
@@ -214,6 +215,7 @@ runMux :: forall m mode.
        -> m ()
 runMux tracer Mux {muxMiniProtocols, muxControlCmdQueue, muxStatus} bearer = do
     egressQueue <- atomically $ newTBQueue 100
+    labelTBQueueIO egressQueue "mux-eq"
```

Comment:
TODO: this should be named <bearer> ++ "mux-eq"

Reply:
Issue [3472](https://github.com/input-output-hk/ouroboros-network/issues/3472).